images/tier1/onlineroom.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="18" width="18" viewBox="0 0 18 18">
```

Comment:
We already have "online-rooms", which is almost identical to this. Probably one should replace the other.